apiExamples/error.html

```diff
@@ -1 +1,3 @@
-<auro-toast variant="error">Toast content goes here.</auro-toast>
\ No newline at end of file
+<auro-button onClick="showToast('#errorToast')">Show error toast</auro-button>
+<br>
```

Comment:
Not a fan of using HTML as layout. I'd rather see inline CSS to create space if needed versus promoting an unwanted pattern. 

Please update all these partial files.